src/packages/@environments/features/environment-explorer/layout/EnvironmentExplorer.svelte

```diff
@@ -30,10 +37,13 @@
    */
   export let onSaveEnvironment;
 
+  export let onFetchEnvironmentGuide;
+  export let onUpdateEnvironmentGuide;
+  let showContainer = false;
+
```

Comment:
can we have a better name for that clarifies what type of container it is?
also use "is" before variable name.